src/mem/ruby/protocol/MOESI_AMD_Base-dir.sm

```diff
@@ -362,6 +362,9 @@ machine(MachineType:Directory, "AMD Baseline protocol")
         if (in_msg.Type == CoherenceResponseType:CPUPrbResp) {
           trigger(Event:CPUPrbResp, in_msg.addr, entry, tbe);
         } else if (in_msg.Type == CoherenceResponseType:CPUData) {
+          //Deprecated - WriteBack data (wb_data) is now sent with VicDirty itself
+          //Leaving this in for reference
+          assert(0);
```

Comment:
I would just remove this. The error below should tell you which message type is missing